src/main/java/io/spring/githubchangeloggenerator/ApplicationProperties.java

```diff
@@ -16,24 +16,26 @@
 
 package io.spring.githubchangeloggenerator;
 
+import static io.spring.githubchangeloggenerator.ApplicationProperties.FormatPlaceholder.*;
+import static io.spring.githubchangeloggenerator.ApplicationProperties.FormatPlaceholder.TITLE;
+
+import io.spring.githubchangeloggenerator.github.service.Repository;
```

Comment:
Please don't re-order the imports. It adds noise to the diff, making the changes harder to review. If you run `./gradlew build`, you will see failures from both Checkstyle and Spring Java Format that should be corrected please.